data_model.md

```diff
@@ -0,0 +1,449 @@
+uploaded. The forecast/observation variables and units that are
+permitted are as follows:
+
+-   surface temperature: degrees Celsius
+
+-   surface wind speed: meters per second
+
+-   Global Horizontal Irradiance: Watts per meter squared
+
+-   Direct Normal Irradiance: Watts per meter squared
+
+-   Diffuse Horizontal Irradiance: Watts per meter squared
+
+-   Plane of array irradiance: Watts per meter squared
+
+-   Relative humidity: %
+
+-   AC power: megawatts
+
+-   DC power: megawatts
+
+-   PDF probability: %
+
+-   CDF value:
+
+Timestamp and timezone formatting is addressed below.
+
+Data Formats
+------------
+{: .anchor}
+
+In this section Data is defined as both observation and forecast data.
+The data contain a time series of data points. Each data point has
+fields determined by the data’s type listed below:
+
+#### Observations
+{: .anchor}
+
+-   Timestamp: A timestamp in ISO-8601 format including a timezone, e.g.
+    2018-12-05T23:01:45-07:00 or 20181206T060145Z.
+
+-   Value: Values for the variable in units defined above.
+
+-   Quality Flag: A flag indicating if the value is questionable. Either
+    0 or 1.
+
+#### Forecasts
+{: .anchor}
+
+Single-valued forecasts (e.g. mean or 50<sup>th</sup> percentile) comprise a
+series of Timestamp, Value pairs in the following format:
+
+-   Timestamp: A timestamp in ISO-8601 format
+    including a timezone, e.g. 2018-12-05T23:01:45-07:00 or
+    20181206T060145Z.
+
+
+-   Value: Values for the variable in units defined above.
+
+Probabilistic forecasts comprise the same set of Timestamp, Value pairs
+as deterministic forecasts *for each probability bin*. To upload a
+complete probabilistic forecast, users upload one forecast per
+probability bin. The metadata of the probabilistic forecast defines the
+*distribution type* (cumulative distribution function (CDF) or
+probability distribution function (PDF) and the *bins*. For a CDF, the
+Value is the quantity of MW, GHI, etc. for the quantile associated with
+each individual forecast upload. For a PDF, the Value is the probability
+corresponding the interval range of MW, GHI, etc. for each individual
+forecast upload. The full set of metadata required to describe
+probabilistic forecasts will be determined later.
+
+### Downloads
+{: .anchor}
+
+
+Data will be available for download from the Solar Forecast Arbiter API
+and dashboard as either CSV or JSON files. Both formats will contain the
+name and id of the forecast/observation requested. In the CSV format,
+this information will be included in the header. In the JSON format,
+these parameters will be keys in the JSON object with the data listed
+under “values”.
+
+#### CSV
+{: .anchor}
+
+```
+# name: Power Plant 1 Power
+# id: testid
+timestamp,value,questionable
+2018-11-22T12:01:48Z,10.23,0
+2018-11-22T12:07:38Z,10.67,0
+```
+
+#### JSON
+{: .anchor}
+
+
+    {
+      “name”: “Power Plant 1 Power”,
+      “id”: “testid”,
+      “_links”: {
```

Comment:
is this supposed to be an underscore?

Reply:
Yes, here the prefixed underscore indicates that the field is not an attribute of the requested object.